Library/.rubocop.yml

```diff
@@ -7,124 +7,126 @@ AllCops:
 
 require: ./Homebrew/rubocops.rb
 
+# enable all formulae audits
 FormulaAudit:
   Enabled: true
 
+# enable all formulae strict audits
 FormulaAuditStrict:
   Enabled: true
 
+# disable all formulae strict audits by default
 NewFormulaAudit:
   Enabled: false
 
 # `system` is a special case and aligns on second argument
 Layout/AlignParameters:
   Enabled: false
 
+# this is a bit less "floaty"
 Layout/CaseIndentation:
   EnforcedStyle: end
 
-Layout/EmptyLineBetweenDefs:
-  AllowAdjacentOneLineDefs: true
-
+# this is a bit less "floaty"
 Layout/EndAlignment:
-  EnforcedStyleAlignWith: variable
-
-Layout/IndentArray:
-  EnforcedStyle: special_inside_parentheses
+  EnforcedStyleAlignWith: start_of_line
 
+# enforce use of <<~EOS
 Layout/IndentHeredoc:
   EnforcedStyle: squiggly
 
 # conflicts with DSL-style path concatenation with `/`
 Layout/SpaceAroundOperators:
   Enabled: false
 
-# favor parens-less DSL-style arguments
+# # favor parens-less DSL-style arguments
 Lint/AmbiguousOperator:
   Enabled: false
 
 # so many of these in formulae and can't be autocorrected
+# TODO: fix these as `ruby -w` complains about them.
 Lint/AmbiguousRegexpLiteral:
   Enabled: false
 
-# favor parens-less DSL-style arguments
+# # favor parens-less DSL-style arguments
 Lint/AmbiguousBlockAssociation:
   Enabled: false
 
 # assignment in conditions are useful sometimes
+# TODO: add parentheses for these and remove
 Lint/AssignmentInCondition:
   Enabled: false
 
+# we output how to use interpolated strings too often
+Lint/InterpolationCheck:
+  Enabled: false
+
 # so many of these in formulae and can't be autocorrected
 Lint/ParenthesesAsGroupedExpression:
   Enabled: false
 
 # most metrics don't make sense to apply for formulae/taps
 Metrics/AbcSize:
   Enabled: false
-
 Metrics/BlockLength:
   Enabled: false
-
 Metrics/ClassLength:
   Enabled: false
-
 Metrics/CyclomaticComplexity:
   Enabled: false
-
-Metrics/LineLength:
-  # 118 is the goal as GitHub diff UI wraps beyond that
-  Max: 189
-  # ignore manpage comments and long single-line strings
-  IgnoredPatterns: ['#: ', ' url "', ' mirror "', ' plist_options :']
-
 Metrics/MethodLength:
   Enabled: false
-
-Metrics/ModuleLength:
+# Metrics/ModuleLength:
+#   Enabled: false
+Metrics/PerceivedComplexity:
   Enabled: false
 
+# keyword arguments don't have the same readability problems
 Metrics/ParameterLists:
   CountKeywordArgs: false
 
-Metrics/PerceivedComplexity:
+# GitHub diff UI wraps beyond 118 characters (so that's the goal)
+Metrics/LineLength:
+  Max: 189
```

Comment:
Wouldn't object to knocking this down a bit, FWIW. I have a file locally of all the core formulae that have overly long lines, which I've started working through. I think 150 is doable fairly quickly.